kedro/framework/project/__init__.py

```diff
@@ -253,4 +253,12 @@ def validate_settings():
     error message ``Expected an instance of `ConfigLoader`, got `NoneType` instead``.
     More info on the dynaconf issue: https://github.com/rochacbruno/dynaconf/issues/460
     """
+    if not PACKAGE_NAME:
```

Comment:
```suggestion
    if PACKAGE_NAME is None:
```
It's almost always slightly preferable to check against `None` like this.